src/Build.UnitTests/BuildEnvironmentHelper_Tests.cs

```diff
@@ -19,6 +20,13 @@ public class BuildEnvironmentHelper_Tests
 #else
         private const string MSBuildExeName = "MSBuild.exe";
 #endif
+
+        [Fact]
+        public void ShouldBeRunningInTests()
```

Comment:
https://github.com/cdmihai/msbuild/blob/ca62c0d4a3d9ba003176c4069a976978c59949d7/src/Build.UnitTests/BuildEnvironmentHelper_Tests.cs#L275-L284

😄

Reply:
Best kind of bugfix. :)